src/indexer/utils/substrate/index.ts

```diff
@@ -116,7 +116,6 @@ export async function saveDeposit(
   if (transfer) {
     const dataTransferToUpdate = {
       depositNonce: Number(depositNonce),
```

Comment:
@mpetrun5 here I have removed the sender, since this is now embedded into the account schema as the address attribute. I'm not quite sure if this is the right thing to do when updating transfers on substrate. Wdyt?

Also, perhaps we can avoid this by maintaining the sender as a top level attr on the transfer schema and the account schema will only have the address status.

Reply:
This is fine.

Reply:
Just check that API now also returns account with address info please so we have that information.